### DISCUSSION TITLE
How to short-circuit the credential delay timeout

### DISCUSSION THREAD

**Discussion by isalkind** [maxileith/homebridge-appletv-enhanced#629] [Q&A] [answered]
I'm getting these messages in my log. I believe this all started as a result of my Apple TV automatically upgrading to tvOS 18. Unfortunately, this originally happened in the middle of the night, so I wasn't standing by to enter the PIN when prompted. And now I'm stuck in this crazy retry loop where I'm never present to enter the PIN. 😀 

What can I reset so that I can get out of this?

```
Credentials are no longer valid. Need to repair ...
You need to pair your Apple TV before the plugin can connect to it. Enter the PIN that is currently displayed on the device here: http://192.168.0.18:42085/
Apple TV Living Room: Too many attempts. Waiting for 10604 seconds before retrying. The webpage to pair the Apple TV will become available again when the plugin is attempting the next pairing attempt.
```

**Comment by becker250** [comment]
I'm also having this issue, trying to see if there's a way to manually pair without the timeout, having to wait 3 hours for the next chance and then it's been so long the log has already deleted that portion to click on the link. Yay

**Comment by isalkind** [comment, marked as answer]
This issue https://github.com/maxileith/homebridge-appletv-enhanced/issues/620 led me to a resolution. I used the `mv` command as specified and restarted the plugin to get back in the game.
